sources/models/src/lib.rs

```diff
@@ -79,6 +79,16 @@ The `#[model]` attribute on Settings and its sub-structs reduces duplication and
 * [Model](src/aws-k8s-1.28-nvidia/mod.rs)
 * [Default settings](src/aws-k8s-1.28-nvidia/defaults.d/)
 
+## aws-k8s-1.29: Kubernetes 1.29
+
+* [Model](src/aws-k8s-1.29/mod.rs)
+* [Default settings](src/aws-k8s-1.29/defaults.d/)
```

Comment:
All the links for the other variants needs to be updated as well to point to `*-k8s-1.29`. The rendered README on github does not follow symlinks.